packages/components/src/input/AddressInput.stories.tsx

```diff
@@ -0,0 +1,16 @@
+import { storiesOf } from "@storybook/react";
+import * as React from "react";
+import { AddressInput } from "./AddressInput";
+
+storiesOf("AddressInput", module)
```

Comment:
Should we put all the form inputs under one "Input" story?

Not a blocker... just a question.